tests/libtest.sh

```diff
@@ -514,8 +514,15 @@ os_repository_new_commit ()
     cd ${test_tmpdir}
 }
 
+assert_has_setfattr () {
+    if ! which setfattr 2>/dev/null; then
+        assert_no_reached "no setfattr available to determine xattr support"
```

Comment:
Missing a `t` in `not` right?  Could just use `fatal`.

Reply:
Thank! This is fixed in #1199.